pytests/test_accomodation.py

```diff
@@ -14,19 +15,31 @@ def testTahi(self):
                 "receivesAccommodationSupport": False
             },
             "threshold": {
-                "isAccommodationSupplement": True
-            },
-            "benefit": {
-                "isStudentAllowance": False
+                "income": {
+                    "AccommodationSupplement": True
+                },
+                "cash": {
+                    "AccommodationSupplement": True
+                }
             }
         }
-        response = self.runReason(body).json()
+
+    def testBasic(self):
+        response = self.runReason(self.basic_body).json()
 
         isAccommodationSupplement = response.get(
             'benefit').get('isAccommodationSupplement')
-        print(isAccommodationSupplement)
-        self.assertTrue(len(isAccommodationSupplement) > 0)
 
+        self.assertEqual(len(isAccommodationSupplement), 3)
+        last_rule = isAccommodationSupplement[-1]
```

Comment:
I'm not sure it needs to be the last rule.
Maybe it's "Any rule that is conclusive".